### PR TITLE
Updating almost all devices to use PMPS database

### PR DESCRIPTION
**Pull request by jyotiphy**
1. Updating motion library
2. Updating cc-lib
3. Updating PRGs of
*IM1K1, IM2K1, IM1K2, IM2K2, IM3K2, IM4K2, IM5K2, IM6K2
*TM1K2, TM2K2
*PF1K2, PF2K2
*LI2K2, LI3K2
4. Also updated database entries for TM1K2, TM2K2
5. Tested the code and seems to be reading database correctly

### CODE REVIEW

File: plc-kfe-rix-motion/kfe_rix_motion/POUs/PRG_IM3K2_PPM.TcPOU [hunk 1]
```diff
@@ -19,40 +19,40 @@ VAR
                               .fbYagThermoCouple.bOverrange := TIIB[IM3K2-EL3314-E4]^TC Inputs Channel 2^Status^Overrange;
                               .fbYagThermoCouple.iRaw := TIIB[IM3K2-EL3314-E4]^TC Inputs Channel 2^Value'}
     fbIM3K2: FB_PPM;
+	fbIM2K2: INT;
+	stPower: INT;
+	stYag1: INT;
```
Comment (ZLLentz) [inline plc-kfe-rix-motion/kfe_rix_motion/POUs/PRG_IM3K2_PPM.TcPOU:24]: Did these get added automatically/accidentally?

File: plc-kfe-rix-motion/kfe_rix_motion/POUs/PRG_IM4K2_PPM.TcPOU [hunk 2]
```diff
@@ -19,40 +19,39 @@ VAR
                               .fbYagThermoCouple.bOverrange := TIIB[IM4K2-EL3314-E4]^TC Inputs Channel 2^Status^Overrange;
                               .fbYagThermoCouple.iRaw := TIIB[IM4K2-EL3314-E4]^TC Inputs Channel 2^Value'}
     fbIM4K2: FB_PPM;
+	fbIM3K2: INT;
+	stPower: INT;
```
Comment (ZLLentz) [inline plc-kfe-rix-motion/kfe_rix_motion/POUs/PRG_IM4K2_PPM.TcPOU:23]: Did these also get added automatically/accidentally?

File: plc-kfe-rix-motion/kfe_rix_motion/POUs/PRG_IM3K2_PPM.TcPOU (hunk 1 above)
Comment (ZLLentz) [reply plc-kfe-rix-motion/kfe_rix_motion/POUs/PRG_IM3K2_PPM.TcPOU:24]:
```suggestion
```

File: plc-kfe-rix-motion/kfe_rix_motion/POUs/PRG_IM4K2_PPM.TcPOU (hunk 2 above)
Comment (ZLLentz) [reply plc-kfe-rix-motion/kfe_rix_motion/POUs/PRG_IM4K2_PPM.TcPOU:23]:
```suggestion
```

**Review by ZLLentz — APPROVED** [review]
I read through this very carefully and there are no blocking issues. It looks good!
I have two suggestions for places where it looks like extra unused variable declarations snuck their way into the code.

Once you are running an updated IOC we can easily check in the transfer GUI that everything loaded properly.

File: plc-kfe-rix-motion/kfe_rix_motion/POUs/PRG_IM3K2_PPM.TcPOU (hunk 1 above)
Comment (jyotiphy) [reply plc-kfe-rix-motion/kfe_rix_motion/POUs/PRG_IM3K2_PPM.TcPOU]: Yes, I think it got added automatically. Removed.

File: plc-kfe-rix-motion/kfe_rix_motion/POUs/PRG_IM4K2_PPM.TcPOU (hunk 2 above)
Comment (jyotiphy) [reply plc-kfe-rix-motion/kfe_rix_motion/POUs/PRG_IM4K2_PPM.TcPOU]: Fixed.

**Pull request merged**